<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTransformerDialog</class>
 <widget class="QDialog" name="ImageTransformerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Transformer</string>
  </property>
  <widget class="QListWidget" name="listWidgetPages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>241</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpsideDown">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上下反転</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonImageTransformer.qrc">
     <normaloff>:/Qtres/Rotate2.png</normaloff>:/Qtres/Rotate2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeftsideRight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>左右反転</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonImageTransformer.qrc">
     <normaloff>:/Qtres/Rotate.png</normaloff>:/Qtres/Rotate.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonImageTransformer.qrc">
     <normaloff>:/Qtres/Close.png</normaloff>:/Qtres/Close.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMasterImage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Master Image</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTargetImage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Image</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ButtonImageTransformer.qrc"/>
 </resources>
 <connections/>
</ui>
